src/types/Movie.ts

```diff
@@ -0,0 +1,7 @@
+export interface Movie {
+  title: string;
+  description: string;
```

Comment:
```suggestion
  description?: string;
```
In task -description is optional. There may be no description on server.